samples/nrf9160/aws_fota/src/main.c

```diff
@@ -520,6 +521,18 @@ static void aws_fota_cb_handler(struct aws_fota_event *fota_evt)
 	}
 }
 
+void erase_modem_update_bank(void)
```

Comment:
Should we add this to aws_fota module and add it to SYS_INIT so that its called before main when the aws_fota library is included?

Reply:
We can't the bsdlib and AT cmd needs to be initialized before calling it. That won't happen if we put it in `SYS_INIT`.